### ISSUE TITLE
Missing header in region file!

### ISSUE THREAD

**Issue by polymetric1** [llbit/chunky#348]
I just downloaded chunky, like 5 minutes ago, maybe even less.
I load up a world (not in my `.minecraft/saves/` directory, rather from my local copy of a friend's server), and it immediately popped up with this error:

![screenshot_2174](https://cloud.githubusercontent.com/assets/13613437/12103144/e696b5ac-b30d-11e5-9980-ab361bf298c7.png)

When I try to close it, it opens thousands more of them.
I googled it, and I haven't found any documentation of this issue, ever.
I searched all the github issues, nothing.

Here's my log: https://gist.github.com/polymetric1/0100f3baaf05b9135cfe

This isn't entirely a huge problem, it's just a bit annoying and I'd like to know what's causing it, if it isn't a problem with Chunky itself, and rather it's my world (which tbh has had a few weird merges over FTP, I wouldn't be surprised if it was kind of screwed up.)

**Comment by empirebuilder1** [comment]
No problem with 15 individual worlds on my computer. I'd say with great certainty that one of your regions is corrupted. 

Perhaps you could .ZIP the world and upload to Dropbox/here if it's not too big, and I can test it.

**Comment by polymetric1** [comment]
I haven't really noticed it doing this lately, and I also happened to have
deleted the spawn region in MCEdit, because they were corrupted. I now know
why this error happened. I think it should be more well documented though,
because I thought there was something wrong with Chunky at first.

**Comment by llbit** [comment]
@polymetric1 any suggestions for how we can improve the documentation for this issue?

**Comment by llbit** [comment]
This was fixed in 3651f9e4c3fe2ebb067ebd487340578b5d44d884

I encountered the issue yesterday while I was loading a map that was currently open in Minecraft. The map loaded fine later. The issue might be caused by reading an inconsistent world state while Minecraft is writing to the world. I also tested introducing random errors like this in the chunk loading to reproduce the issue.

I have changed the way this type of error is handled. Now the error messages for failing to load a region or a chunk are reported on the STDERR stream instead, so the application won't be effectively disabled by thousands of warning messages. I also made sure that after a chunk has failed to load Chunky won't attempt to re-load it again immediately (causing an infinite loop of errors).
